tst/EnergyPlus/unit/HVACUnitarySystem.unit.cc

```diff
@@ -2388,6 +2582,10 @@ TEST_F(EnergyPlusFixture, UnitarySystemSizingTest_ConfirmUnitarySystemSizingTest
     DataEnvironment::StdRhoAir = 1000; // Prevent divide by zero in ReportSizingManager
 
     UnitarySystem.allocate(HVACUnitarySystem::NumUnitarySystem);
+    MultiOrVarSpeedCoolCoil.allocate(HVACUnitarySystem::NumUnitarySystem);
+    MultiOrVarSpeedCoolCoil = false;
+    MultiOrVarSpeedHeatCoil.allocate(HVACUnitarySystem::NumUnitarySystem);
+    MultiOrVarSpeedHeatCoil = false;
```

Comment:
The code now only initializes the idle flow rate (based on this variable) if it is a multi-speed coil so this variable must be allocated in unit tests.